GitExtensions/GitExtensions.csproj

```diff
@@ -21,6 +21,10 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <PackageReference Include="DotnetRuntimeBootstrapper">
```

Comment:
We need to package this into the installer (.\Setup\Product.wxs). You should be able to build the installer using `msbuild` (from the develop prompt), e.g. 
```
Setup> msbuild
```

Reply:
I have run the command, but seems to fail, any ideas?

![image](https://user-images.githubusercontent.com/20372363/129474311-79734fb9-5efe-434f-994e-d8ca1e675460.png)

Reply:
Also, what about just selecting "Release" in Visual Studio and building from there? I have used this for testing in a Windows VM.